python/lsst/synpipe/compareModel.py

```diff
@@ -46,28 +47,28 @@ def writeNumpyTable(fakeTable):
     """Writes output to numpy
     """
     npTable = np.recarray(len(fakeTable),
-                          dtype={'names':['id', 'fakeid', 'visit', 'ccd',
-                                          'cmodelMag', 'expMag',
-                                          'devMag',
-                                          'expReff', 'devReff',
-                                          'expBA', 'devBA',
-                                          'expPosAng', 'devPosAng',
-                                          'cmodelMagErr', 'expMagErr',
-                                          'devMagErr', 'nchild', 'parent'],
-                                 'formats':[long, int, int, int,
-                                            float, float,
-                                            float, float, float, float,
-                                            float, float, float, float,
-                                            float, float, int, long]})
+                          dtype={'names': ['id', 'fakeid', 'visit', 'ccd',
+                                           'cmodelMag', 'expMag',
+                                           'devMag',
+                                           'expReff', 'devReff',
+                                           'expBA', 'devBA',
+                                           'expPosAng', 'devPosAng',
+                                           'cmodelMagErr', 'expMagErr',
+                                           'devMagErr', 'nchild', 'parent'],
+                                 'formats': [int, int, int, int,
```

Comment:
Shouldn't  ```id``` and ```parent``` stay as ```long``` instead of being changed to ```int``` ?

Reply:
In python3 there is no `long`. In python2 64-bit an `int` is 64-bit so it doesn't make a difference there either. I think the only reason to use `long` in python2.7 is if you are building on 32-bit which we aren't.